misc_urls.py

```diff
@@ -105,7 +105,7 @@
         # Security report URL:
         "<security_report_url>",
         # Logging URL:
-        "<logging_url>",
+        "http://167.71.78.68:3000/explore?orgId=1&left=%7B%22datasource%22:%22Loki%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%22,%22queryType%22:%22range%22%7D%5D,%22range%22:%7B%22from%22:%22now-1h%22,%22to%22:%22now%22%7D%7D",
```

Comment:
i can't login with the user/pass agreed upon on Teams. is this on purpose?

Reply:
Turns out anonymous viewers can't access the explore page, so I've added a user with the provided credentials now.